### PR TITLE
Replace Reorderable list component

### PR DESCRIPTION
**Pull request by DilwoarH**
## What

Replace reorderable component with the global version in govuk publishing gem

## Why

This will clean up code and keep it consistent.

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

https://trello.com/c/9CQm5A0k/251-update-publishing-tools-to-use-new-reorderable-component

### CODE REVIEW

File: spec/support/coronavirus_feature_steps.rb [hunk 1]
```diff
@@ -451,7 +451,7 @@ def stub_discard_coronavirus_page_no_draft
   end
 
   def i_see_subsection_one_in_position_one
-    expect(find(".app-c-reorderable-list:first-child")).to have_content "I am first"
```
Comment (kevindew) [inline spec/support/coronavirus_feature_steps.rb:454]: Can this commit be part of the previous one where tests are already being fixed?

**Review by kevindew — COMMENTED** [review]
This looks good to me 👍 

It looks like we still need to remove sortableJS from package.json and yarn.lock.

**Comment by DilwoarH** [comment]
> This looks good to me 👍
> 
> It looks like we still need to remove sortableJS from package.json and yarn.lock.

good spot

File: spec/support/coronavirus_feature_steps.rb (hunk 1 above)
Comment (DilwoarH) [reply spec/support/coronavirus_feature_steps.rb:454]: this was meant to be fixed up - let me do it now

**Comment by DilwoarH** [comment]
> This looks good to me 👍
> 
> It looks like we still need to remove sortableJS from package.json and yarn.lock.

done

**Review by kevindew — APPROVED** [review]
Super ⭐

**Comment by DilwoarH** [comment]
> Super ⭐

Thank you!

**Pull request merged**